templates/hotgraphicLayoutPins.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { classes } from 'core/js/reactHelpers';
+
+export default function HotgraphicLayoutPins(props) {
+
+  const {
+    _items,
+    _graphic,
+    _useNumberedPins,
+    onPinClicked
+  } = props;
+
+  const itemAriaLabel = (index, pin, title) => {
```

Comment:
Rather than having the itemAriaLabel outside of the item, could we add this inside the _items.map (as with the accordion), something like the below:  

const ariaLabel = `${_isVisited ? visited + '. ' : ''} ${_useNumberedPins ? (index + 1) + '.' : ''} ${compile(title)}. Item ${(index + 1)} of ${_items.length}`;